functions/src/index.ts

```diff
@@ -31,3 +31,49 @@ export const getSPLFunction = SPLFunction.getSPL;
 export const getBulletinFunction = bulletinFunction.getBulletin;
 export const getAndroidVersionFunction = androidVersionFunction.getAndroidVersion;
 
+//converter
+import * as data from './bulletin.json';
+
+const SPLsArray : any[] = [];
+const found : string[] = [];
+ for(const vulnerabilities of data.vulnerabilities){
+    const patchLevel = vulnerabilities.patch_level;  
+    if (found.indexOf(patchLevel) === -1) {
+        found.push(patchLevel);
+        SPLsArray.push(patchLevel);
+    }
+}
+const SPLs = {
+    'SPLs': SPLsArray
+ };
+const ASBs = { [data.ASB] : SPLs };
+const Bulletin_SPLs = {'ASBs': ASBs};
+
+const pubishDate = data.published;
+const result : Record<string, object> = {};
+for(const spl of SPLsArray){
+    const cveIDsArray : string[] = [];
+    const splDetails = {
+        'Publish_Date': pubishDate,
+        'CVE_IDs': cveIDsArray
+     };
+     for(const vulnerabilities of data.vulnerabilities){
+        const patchLevel = vulnerabilities.patch_level;
+        if(patchLevel === spl){
+            const cveID = vulnerabilities.CVE;
+            splDetails.CVE_IDs.push(cveID);
+        }
+        result[spl] = splDetails;
+      }
+}
+
+const SPL_CVE_IDs = {'SPLs' : result};
+const output = {SPL_CVE_IDs, Bulletin_SPLs};
```

Comment:
the conversion code ideally should not go in the index.ts for the functions, this should be in a separate cli module